### DISCUSSION TITLE
Can't get cheatsheet to show

### DISCUSSION THREAD

**Discussion by greehny** [end-4/dots-hyprland#555] [Q&A]
Hey all,

Just wanted to start by saying this is my first time using arch, hyprland and these dotfiles so please bare with me.

I'm currently having issues with opening the built-in cheatsheet using Super + "/". It just won't show at all.

I've already researched a bit into some causes, mainly with ags. Though ags runs fine, and even after I restart it I can't get the cheatsheet to show. Everything else works like the sidebar and whatnot, but it is just the cheatsheat that won't show

Any ideas?

**Comment by end-4** [comment]
did you install everything or just copied ags folder? the keybind is available if hyprland is configured to have it

**Reply by greehny** [reply]
Thank you for your reply!

I used the install script for the majority of my setup, then had to build gradience-git myself using an old version of python-cssutils.

Everything else I let the script do.

**Reply by end-4** [reply]
"the majority" is vague
but if you said yes to the installation of hyprland config it should be fine
ugh
maybe in a terminal run `pkill ags; ags`, hit the keybind, then send logs if any
